### ISSUE TITLE
[meta][meta] Refactor meta bugs

### ISSUE THREAD

**Issue by mcomella** [mozilla-mobile/firefox-tv#351]
Issues coming from [estimation breakdown](https://docs.google.com/document/d/1SLgyJtBtAOGiHpudHZpmJB85dap4ZbqIhN-_Cpz0gOk/edit#):
- [ ] #346: MainActivity
- [ ] #349: Settings
- [ ] #350 Onboarding
- [ ] #353 Browser
- [ ] #416 Tests (fixed with #390?)
- [ ] #392 CI
- [ ] #445 l10n
- [ ] Post tasks (e.g. rm unused resources)

Followed by:
- 2.0 features
- Nice-to-haves

**Comment by mcomella** [comment]
Adding to milestone not because we'll finish it but so we can keep track of this meta bug.

**Comment by mcomella** [comment]
Blocked on post 2.0 work.

**Comment by mcomella** [comment]
Will break out.
